rfc/github-org-migration.md

```diff
@@ -0,0 +1,773 @@
+    - It SHOULD NOT be migrated.
+- [ ] [dag.w3s.link](https://github.com/web3-storage/dag.w3s.link)
+    - Trustless GW for Saturn.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [w3admin](https://github.com/web3-storage/w3admin)
+    - Admin interface for the new UCAN APIs.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [gendex-consumer](https://github.com/web3-storage/gendex-consumer)
+    - It can be archived as it didn't quite work out.
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [content-claims](https://github.com/web3-storage/content-claims)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [sync-multihash-sha2](https://github.com/web3-storage/sync-multihash-sha2)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [hoverboard](https://github.com/web3-storage/hoverboard)
+    - Bitswap in CF workers used in production.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [access-proxy](https://github.com/web3-storage/access-proxy)
+    - [ ] Ask @travis if it can be archived.
+- [ ] [RFC](https://github.com/web3-storage/RFC)
+    - It SHOULD NOT be archived.
+    - It SHOULD be migrated.
+- [ ] [loki-tail-worker](https://github.com/web3-storage/loki-tail-worker)
+    - Worker to tail logs.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [content-claims-gql](https://github.com/web3-storage/content-claims-gql)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [sha256it](https://github.com/web3-storage/sha256it)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [migrate-block-index](https://github.com/web3-storage/migrate-block-index)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [fr32-sha2-256-trunc254-padded-binary-tree-multihash](https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash)
+    - Lib used for piece CIDs.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [migrate-block-index-infra](https://github.com/web3-storage/migrate-block-index-infra)
+    - It SHOULD be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [w3stat](https://github.com/web3-storage/w3stat)
+    - To diagnose CID problems.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [go-ucanto](https://github.com/web3-storage/go-ucanto)
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [pieces](https://github.com/web3-storage/pieces)
+    - Cli for devs.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [console](https://github.com/web3-storage/console)
+    - Web UI for w3up.
+    - It SHOULD NOT be archived.
+    - It SHOULD NOT be migrated.
+- [ ] [one-webcrypto](https://github.com/web3-storage/one-webcrypto)
+    - Fork
+    - [ ] Ask @gobengo if it can be deleted or archived.
```

Comment:
`@web3-storage/access` (part of w3up-client) depends on this https://github.com/web3-storage/w3up/blob/6fa7797ba6f5c0bbd359c807e257882a7d5d6fb8/packages/access-client/package.json#L113

Reply:
thank you 🙏 . great to know